resources/views/components/filter/disclosure_modal.blade.php

```diff
@@ -0,0 +1,10 @@
+<div v-if="{{ $attributes->get('v-if') }}"
```

Comment:
This attribute selection is a bit of a gotcha... But the syntax works, so let's try :)

I'd just suggest following the convention of passing (almost) all attributes to the root element:
```suggestion
<div v-if="{{ $attributes->except('@close')->merge(...) }}"
```